templates/taxi/car_list.html

```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form class="form-inline" action="" method="get">
+    {{ car_search|crispy }}
+    <input type="submit" class="btn btn-primary" value="Search">
+  </form>
```

Comment:
Create a separate template for search form and include it where you need to avoid code duplication